fury/tests/test_utils.py

```diff
@@ -865,6 +865,43 @@ def test_color_check():
     npt.assert_equal(global_opacity, 1)
 
 
+def test_check_color_range():
+    # Test input None in color
+    none_color = None
+    assert utils.check_color_range(none_color) == None
+
+    # Test input data
+    valid_color_array = np.array([[0.1, 0.2, 0.3, 0.4], [0.4, 0.5, 0.6, 0.7]])
+    invalid_color_array = [[0.1, 0.2, 1.0, 0.3], [255, 255, 255, 0.7]]
+    invalid_color_array_expected = np.array(
+        [[0.1, 0.2, 1.0, 0.3], [1.0, 0.0, 0.0, 1.0]])
+
+    # Test for valid input
+    npt.assert_array_equal(utils.check_color_range(
+        valid_color_array), valid_color_array)
+
+    # Test for invalid input
+    npt.assert_array_equal(utils.check_color_range(
+        invalid_color_array), invalid_color_array_expected)
+
+    # Test for input of type tuple
+    color_tuple = (0.1, 0.2, 0.3)
+    color_tuple_expected = (0.1, 0.2, 0.3)
+    assert utils.check_color_range(color_tuple) == color_tuple_expected
+
+    # Test for input of type list (invalid)
+    color_list = [100, 150, 200, 0.4]
```

Comment:
That's a good idea to normalize [0, 255] to [0, 1], I will do it.

The initiation is to take care of all out of bounds even more than 255 and less than 0, then give a red color to let the process finish and show a hint and warning to the user. This will also skip the modification of all actor docstring, saying color parameter accept both [0,1] and [0,255].

But I agree normalization is more user friendly and the value more than 255 can be taken care by other function that process rgb values.

Reply:
`normalize_color()` is done.